Source/WebKit/UIProcess/API/glib/WebKitWebsiteDataManager.cpp

```diff
@@ -403,6 +408,22 @@ static void webkit_website_data_manager_class_init(WebKitWebsiteDataManagerClass
             _("The maximum size of local storage in bytes"),
             1, G_MAXUINT, 5 * 1024 * 1024,
             static_cast<GParamFlags>(WEBKIT_PARAM_WRITABLE | G_PARAM_CONSTRUCT_ONLY)));
+
+    /**
+     * WebKitWebsiteDataManager:per-origin-storage-quota:
+     *
+     * Max size of origin data storage. This includes CacheStorage and IndexedDB.
+     * Value of 0 means no change so whatever the default value is it will be used.
```

Comment:
It would be better if the default value of this property was 5*1024*1024 explicitely, like the localStorageQuota one. And if the app sets a different value then that one is used.

Reply:
I used 0 to preserve current Webkit quota that is 1000MB (as noted blow) as I don't want to change the default here. Maybe I can get the default from StorageQuotaManager::defaultQuota() directly here.